app/javascript/components/Calendar/__snapshots__/test.js.snap

```diff
@@ -1,31 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`loads 1`] = `
-<Layout
-  currentPath="/"
->
-  <Uncontrolled(Calendar)
-    components={
-      Object {
-        "event": [Function],
-        "toolbar": [Function],
-      }
-    }
-    eventPropGetter={[Function]}
-    events={Array []}
-    onNavigate={[Function]}
-    onSelectEvent={[Function]}
-    popup={true}
-    style={
-      Object {
-        "paddingBottom": 20,
-      }
-    }
-    views={
-      Array [
-        "month",
-      ]
-    }
-  />
-</Layout>
+<Consumer>
+  <Component />
+</Consumer>
```

Comment:
This probably happened when wrapping the calendar component with `withNamespaces`